packages/esm-patient-search-app/src/compact-patient-search-extension/index.tsx

```diff
@@ -65,13 +65,7 @@ const CompactPatientSearchComponent: React.FC<CompactPatientSearchProps> = ({
     inputRef.current.focus();
   }, [inputRef]);
 
-  const focussedResult = useArrowNavigation(
-    inputRef,
-    patients?.length ?? 0,
-    handlePatientSelection,
-    handleFocusToInput,
-    -1,
-  );
+  const focussedResult = useArrowNavigation(patients?.length ?? 0, handlePatientSelection, handleFocusToInput, -1);
```

Comment:
Since this variable was introduced in a previous PR by me, and is also being used at many places, I believe we can skip this change for now, but yes, I'll do keep this in mind in future. I hope that'd be okay.
Thanks!